input/ez.raml

```diff
@@ -10,6 +10,7 @@ baseUri: https://ezsystems.github.io/ezplatform-rest-reference
 
 /content:
     displayName: Managing content
+    /assets: !include ez-content-assets.raml
```

Comment:
Not sure if it influences the order, but I'd move this someplace below and leave content as the first section to display

Reply:
I thought endpoints were ordered alphabetically, hence this location. Let's see what Dawid says